helpers/utils.js

```diff
@@ -0,0 +1,14 @@
+// import { swal } from 'sweetalert';
+
+// import { SUCCESS, ERROR } from '@/constants.js';
+
+// const triggerToast = (type) => {
+//   swal($nuxt.$t('alert.message.error'), {
+//     buttons: false,
+//     className: 'swal-basic ' + type === SUCCESS ? 'success' : 'error',
+//     iconHtml: '<img src="https://picsum.photos/100/100">',
+//     timer: 3000,
+//   });
+// };
+
+// export default triggerToast;
```

Comment:
Can we please remove this if not needed ?

Reply:
resolved